cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -58,6 +58,13 @@ data NodeConfiguration
          -- Node parameters, not protocol-specific:
        , ncSocketPath     :: !(Maybe SocketPath)
        , ncDiffusionMode  :: !DiffusionMode
+         -- | Whether to limit the negotiated node-to-node and node-to-client
+         -- versions to that of the latest official release
+         --
+         -- Defaults to 'False'. For example, the QA Team must set this to
+         -- 'True' in order to exercise a feature that is (partially) developed
+         -- but not scheduled for activation in the next release.
+       , ncEnableDevelopmentalVersions :: !Bool
```

Comment:
`ncEnableDevelopmentVersions` sounds better for me, but I am not native speaker.  Should I adjust my ear? Or both versions are acceptable?

Reply:
@coot I was having a hard time choosing between those two. I think both are OK, neither is great.

I chose "developmental version" because "development version" feels slightly more ambiguous: I think it's a bit more likely that someone would unfortunately interpret "development version" as "the version I should use for development" or something like that. Even though -- in a vague sense -- that's not wrong.

But: I don't think either name is great. Do you have any fresh ideas for something that's clear?

  * experimental version
  * testing version
  * quality assurance version
  * in-development version
  * etc?

Reply:
I like both `in-development` and `experimental` version, another one could be `future-version`.  Maybe the most explicit is `experimental`, what do you think?

Reply:
I just pinged Duncan and he expressed a preference for "Test*", since that's how this flag's ancestors have been named.

So I'm thinking `TestInDevelopmentVersion`.

(PS - Woohoo Mary hard fork!)